Frontend/src/data/customer/columns.js

```diff
@@ -7,6 +7,16 @@ import { activeRentalsForCustomerSelector } from "../selectors";
 
 const backgroundColor = async (customer) => customer.highlight;
 
+async function countRentals(customer_id) {
+  const selectors = [
+    {
+      customer_id: customer_id,
+    },
+  ];
+  return await Database.countDocs(selectors);
```

Comment:
Instead of awaiting a promise inside an async funtion and then returning it you could just return the promise from a normal function. This would behave the same way but require less code.

Reply:
Ok, so without the await it would be equivalent? I still don't quite get the async/await stuff. it's the same in python, it just behaves against my logic :D